habit/src/styles/login.js

```diff
@@ -0,0 +1,11 @@
+const React = require('react-native');
```

Comment:
Is this how this is done commonly?
As in, why do we need require?

Can we just say 
- import {Spreadsheet} from 'react-native';

Reply:
yeah right

Reply:
and, not spreadsheet .. stylesheet :p